server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -925,6 +927,30 @@ def duplicate(
         )
         return StepContext(query, columns + [step.new_column_name])
 
+    @classmethod
+    def _interval_to_seconds(cls, value: Selectable) -> functions.Function:
+        raise NotImplementedError(
+            f"[{cls.DIALECT}] Converting an INTERVAL to seconds is not supported"
+        )
```

Comment:
I hesitated because I thought this could be common to all translators with one or two specializations, but they're all too different. So you're right, I'll make it abstract :slightly_smiling_face:

Reply:
[053c872](https://github.com/ToucanToco/weaverbird/pull/1878/commits/053c872a474d211988f4061077d256c6e4f5ab39)